{"uid":"4b4757e66a1912dae1a509f688f20b0f","name":"categories","children":[{"name":"Product defects","children":[{"name":"AssertionError: [AssertionError] during step: Update user role on a project via CLI\nMessage:  Expected: ERROR: Illegal argument(s) (Last project admin cannot be demoted), got: Your root certificates are out of date.\nYou are using certifi 2025.7.14, however 2025.8.3 is available.\nPlease consider upgrading certifi package, e.g.:\n    python -m pip install --upgrade certifi\nor\n    conda update certifi\n\nERROR: Illegal argument(s) (Last project admin cannot be demoted)","children":[{"name":"Admin demote himself to Manager in project via CLI","uid":"6463e5aa1f9794ad","parentUid":"3d8a2ad6301fe76f3bc93b283d1bcc76","status":"failed","time":{"start":1756086332608,"stop":1756086358270,"duration":25662},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":2,"retriesStatusChange":false,"parameters":[],"tags":["asyncio"]}],"uid":"3d8a2ad6301fe76f3bc93b283d1bcc76"},{"name":"AssertionError: [AssertionError] during step: Remove user from project via CLI\nMessage:  Expected: ERROR: Illegal argument(s) (Last project admin cannot be removed), got: Your root certificates are out of date.\nYou are using certifi 2025.7.14, however 2025.8.3 is available.\nPlease consider upgrading certifi package, e.g.:\n    python -m pip install --upgrade certifi\nor\n    conda update certifi\n\nERROR: Illegal argument(s) (Last project admin cannot be removed)","children":[{"name":"Admin removes himself from project via CLI","uid":"be84f7084ee1afc0","parentUid":"4ddb5be6cfdb5fe6d784d53322499d86","status":"failed","time":{"start":1756087503170,"stop":1756087537675,"duration":34505},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":2,"retriesStatusChange":false,"parameters":[],"tags":["asyncio"]}],"uid":"4ddb5be6cfdb5fe6d784d53322499d86"},{"name":"AssertionError: [AssertionError] during step: <<User3>> :: Login via UI\nMessage: [AssertionError] during step: <<User3>> :: Verify that Welcome new user page displayed\nMessage: Locator expected to be visible\nActual value: <element(s) not found> \nCall log:\n  - Expect \"to_be_visible\" with timeout 5000ms\n  - waiting for locator(\"h3.truncate.text-h3\").filter(has_text=\"Welcome, regression-obik8ouwg1@apolo.us\")","children":[{"name":"User removes User from org via **CLI**","uid":"e0ed28f1bfc5d69e","parentUid":"3909c20f29dff07dfdb24bfd6c50acf9","status":"failed","time":{"start":1756088626407,"stop":1756088689744,"duration":63337},"flaky":true,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":2,"retriesStatusChange":false,"parameters":[],"tags":["asyncio"]},{"name":"Verify Manager can remove User from organization","uid":"853e7c209b064c35","parentUid":"3909c20f29dff07dfdb24bfd6c50acf9","status":"failed","time":{"start":1756089216704,"stop":1756089261995,"duration":45291},"flaky":true,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":2,"retriesStatusChange":false,"parameters":[],"tags":["asyncio"]},{"name":"Verify Manager can remove another Manager from organization","uid":"c00c6d96d6daa82e","parentUid":"3909c20f29dff07dfdb24bfd6c50acf9","status":"failed","time":{"start":1756089443836,"stop":1756089501460,"duration":57624},"flaky":true,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":2,"retriesStatusChange":false,"parameters":[],"tags":["asyncio"]},{"name":"Verify Manager cannot remove Admin from organization","uid":"4b244d764f4859a0","parentUid":"3909c20f29dff07dfdb24bfd6c50acf9","status":"failed","time":{"start":1756089652912,"stop":1756089722462,"duration":69550},"flaky":true,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":2,"retriesStatusChange":false,"parameters":[],"tags":["asyncio"]}],"uid":"3909c20f29dff07dfdb24bfd6c50acf9"},{"name":"AssertionError: [AssertionError] during step: Add user not in organization to project via CLI\nMessage: Expected: \nERROR: User 'regression-org-qwo6oah819v9/regression-npsrn53usw' not found in cluster 'default' \nbut got \nYour root certificates are out of date.\nYou are using certifi 2025.7.14, however 2025.8.3 is available.\nPlease consider upgrading certifi package, e.g.:\n    python -m pip install --upgrade certifi\nor\n    conda update certifi\n\nERROR: User 'regression-org-qwo6oah819v9/regression-npsrn53usw' not found in cluster 'default'","children":[{"name":"Add user not in organization to project via CLI","uid":"7e37b2d9abea0c09","parentUid":"58c0e17f8c8b957e61a02be67a3ab677","status":"failed","time":{"start":1756089851427,"stop":1756089914380,"duration":62953},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":2,"retriesStatusChange":false,"parameters":[],"tags":["asyncio"]}],"uid":"58c0e17f8c8b957e61a02be67a3ab677"}],"uid":"8fb3a91ba5aaf9de24cc8a92edc82b5d"}]}